lib/pavilion/test_config/file_format.py

```diff
@@ -410,6 +458,21 @@ class TestConfigLoader(yc.YamlConfigLoader):
                               "script. Added to the beginning of the run "
                               "script. These are generally expected to "
                               "be host rather than test specific."),
+                yc.KeyedElem(
+                    'spack', elements=[
+                        yc.ListElem(
+                            'install', sub_elem=yc.StrElem(),
+                            help_text='The list of spack packages to be '
+                                      'installed.'
+                        ),
```

Comment:
I don't like this, except in the case of the test being a build.

I want to talk about this in Monday's meeting.